### ISSUE TITLE
Usage fails when no static qrc is selected. Is that still a useful option?

### ISSUE THREAD

**Issue by emmenlau** [cneben/QuickQanava#194]
With current QuickQanava 2.2.0 we get the same error as https://github.com/cneben/QuickQanava/issues/92, when we set cmake option `-DQUICK_QANAVA_BUILD_STATIC_QRC=FALSE`.

I tried to follow this up, and find a valid way to get QuickQanava working with this option turned off, but I failed. From what I could see, Qt does not initialize the QuickQanava resources correctly when the static qrc is set to false. I think this may be because the actual C++ library `libQuickQanava.a` is always set to `STATIC`, and in turn the qrc should maybe also be static, is that possible?

Basically my question is this: Is it even sensible to have option `-DQUICK_QANAVA_BUILD_STATIC_QRC=FALSE` when the library is built as static, which is currently enforced in cmake? Because for me this does not work. With `-DQUICK_QANAVA_BUILD_STATIC_QRC=TRUE` on the other hand, everything works just fine!

Does that mean that option `QUICK_QANAVA_BUILD_STATIC_QRC` is just deprecated and should be removed? Or am I just unable to use it wisely?

**Comment by emmenlau** [comment]
Hey, anyone?

**Comment by cneben** [comment]
Sorry for the "long" delay ! Option removed, there is still a qrc until I go with full CMake resources management (next pure Qt6 version 2.5.0 imminent, full CMake configuration not until 2.6.0). Closing.

**Comment by emmenlau** [comment]
Thanks @cneben , sounds very good!
